src/Php80/bootstrap.php

```diff
@@ -19,4 +19,8 @@ function fdiv($dividend, $divisor) { return p\Php80::fdiv($dividend, $divisor);
     if (!defined('FILTER_VALIDATE_BOOL')) {
         define('FILTER_VALIDATE_BOOL', FILTER_VALIDATE_BOOLEAN);
     }
+
+    if (!function_exists('get_debug_type')) {
+        function get_debug_type($value) { return is_object($value) ? get_class($value) : gettype($value); }
```

Comment:
I don't think `gettype` is correct, since `get_debug_type` will return `int` instead of `integer`, looking at the implementation.